README.md

```diff
@@ -109,11 +109,9 @@ Additional resources:
 ## Information for Developers 
 
 * [Virtual Schema API Documentation][vs-api]
-* [Developing and Testing an SQL Dialect](doc/development/developing-sql-dialect/developing_a_dialect.md)
-* [Step-by-step guide to writing your own SQL dialect](doc/development/developing-sql-dialect/step_by_step_guide_to_writing_your_own_dialect.md)
-* [Remote Debugging](doc/development/remote_debugging.md)
-* [Troubleshooting](doc/development/troubleshooting.md)
-* [Versioning](doc/development/versioning.md)
+* [Developing and Testing an SQL Dialect][developing-dialect]
+* [Step-by-step guide to writing your own SQL dialect][step-be-step-developing-guide]
+* [Remote Debugging][remote-debugging]
```

Comment:
I would prefer to remove the content of this section and replace it with a line like:
`For implementing your own JDBC Virtual Schema follow [these guidelines](https://github.com/exasol/virtual-schema-common-jdbc#information-for-developers)`

Or something similar.